### DISCUSSION TITLE
The domains or services to interact with when using GitHub Copilot

### DISCUSSION THREAD

**Discussion by ghost** [community/community#85007] [Copilot Conversations] [answered]
### Select Topic Area

Question

### Body

Our company's network egress direction is configured with the security group and firewall rules，this prevented us from working with GitHub Copilot.
We want to know which domains or services should be allowed when using GitHub Copilot, Then add the domain to the whitelist of the gateway egress.

**Comment by Akash1134** [comment, marked as answer]
Hi @qiaozhi92 

Here's a [detailed troubleshooting guide](https://docs.github.com/en/copilot/troubleshooting-github-copilot/troubleshooting-firewall-settings-for-github-copilot) you may refer to configure your network settings correctly with GitHub Copilot or Copilot Chat. 

Let us know if that helps.
